WalletWasabi.Gui/Converters/VersionStringConverter.cs

```diff
@@ -0,0 +1,30 @@
+using Avalonia.Data.Converters;
+using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Text;
+using WalletWasabi.Exceptions;
+
+namespace WalletWasabi.Gui.Converters
+{
+	public class VersionStringConverter : IValueConverter
+	{
+		public object Convert(object value, Type targetType, object parameter, CultureInfo culture)
+		{
+			if (value is Version version)
+			{
+				// When the amount starts with a '~' then Max is selected
```

Comment:
Is this comment correct?

Reply:
No, lol, that was copypaste from another converter xD